static/js/app.js

```diff
@@ -0,0 +1,286 @@
+$(document).ready(function() {
+    updateCart();
```

Comment:
if you are using vanilla javascript to develop the project then you are not supposed to use Jquery

Reply:
Addressed.